spec/controllers/api/env_settings_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::EnvSettingsController do
+  include ControllerHelpers
+
+  context 'GET index' do
+
+    it 'returns env_settings with keycloak enabled' do
+
+      sentry_env_vars
+
+      get :index
+
+      expect(json['sentry']).to eq('123456')
+    end
+  end
+end
+
+def sentry_env_vars
```

Comment:
maybe rename this to `prepare_sentry_env_vars`? and put it inside a `before(:each)`?